src/main/java/ir/tehran/jug/projects/eventmanagement/web/telegram/bot/EventManagerBot.java

```diff
@@ -0,0 +1,130 @@
+package ir.tehran.jug.projects.eventmanagement.web.telegram.bot;
+
+import org.telegram.telegrambots.bots.TelegramLongPollingBot;
+import org.telegram.telegrambots.meta.api.methods.ParseMode;
+import org.telegram.telegrambots.meta.api.methods.send.SendMessage;
+import org.telegram.telegrambots.meta.api.objects.CallbackQuery;
+import org.telegram.telegrambots.meta.api.objects.Message;
+import org.telegram.telegrambots.meta.api.objects.Update;
+import org.telegram.telegrambots.meta.api.objects.replykeyboard.InlineKeyboardMarkup;
+import org.telegram.telegrambots.meta.api.objects.replykeyboard.ReplyKeyboardMarkup;
+import org.telegram.telegrambots.meta.api.objects.replykeyboard.buttons.InlineKeyboardButton;
+import org.telegram.telegrambots.meta.api.objects.replykeyboard.buttons.KeyboardButton;
+import org.telegram.telegrambots.meta.api.objects.replykeyboard.buttons.KeyboardRow;
+import org.telegram.telegrambots.meta.exceptions.TelegramApiException;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class EventManagerBot extends TelegramLongPollingBot {
+
+    public EventManagerBot(String botToken) {
+        super(botToken);
+    }
+
+    @Override
+    public void onUpdateReceived(Update update) {
+
+        if (update.hasMessage()) {
+            Message message = update.getMessage();
+
+            if (message.hasText()) {
+                String text = message.getText();
+
+                if (text.equals("/start")) {
+                    SendMessage sendMessage = new SendMessage();
+                    sendMessage.setText("\"به برنامه مدیریت ایونت JUG خوش آمدید \uD83D\uDC4B\uD83D\uDE0D\"");
+                    sendMessage.setParseMode(ParseMode.MARKDOWN);
```

Comment:
What is parse mode?

Reply:
> What is parse mode?

The three parse modes in Telegram bots, namely Markdown, MarkdownV2, and HTML, are used to format the text sent by the bot. Here's a breakdown of each parse mode:

Markdown: This parse mode allows you to format text using simple Markdown syntax. You can use asterisks (*) or underscores (_) to make text bold, and use backticks () to create inline code blocks. For example, bold text` will result in bold text.

MarkdownV2: This parse mode is an extended version of Markdown and provides additional formatting options. In addition to the features available in Markdown, you can use double underscores (__) to underline text and tilde (~) to create strikethrough text. For example, __underlined text__ will display as underlined.

HTML: This parse mode allows you to use HTML tags to format text. You can use tags like <b> for bold, <i> for italic, <u> for underline, and <s> for strikethrough. For example, <b>bold text</b> will show as bold text.

Reply:
https://core.telegram.org/bots/api#formatting-options